edx/edx-app-ios
# Follow Dynamic Type Text Size Settings for Local HTML Files

### Description

[LEARNER-6731](https://openedx.atlassian.net/browse/LEARNER-6731)

Mobile is using three local HTML files (MobileAppEula.htm, PrivacyPolicy.htm & TermsOfServices.htm), this PR is adding dynamic type support for these files.

### Notes

### How to test this PR
- Open Login screen or Create Your Account screen
- Open Licence agreement or Term of services or Privacy Policy, the font should update as per accessibly setting.
